astropy/modeling/powerlaws.py

```diff
@@ -15,8 +15,65 @@
 from .parameters import Parameter, InputParameterError
 from ..units import Quantity
 
-__all__ = ['PowerLaw1D', 'BrokenPowerLaw1D', 'SmoothlyBrokenPowerLaw1D',
-           'ExponentialCutoffPowerLaw1D', 'LogParabola1D']
+__all__ = ['PowerLaw1D', 'Beta1D', 'BrokenPowerLaw1D',
+           'SmoothlyBrokenPowerLaw1D', 'ExponentialCutoffPowerLaw1D',
+           'LogParabola1D']
+
+
+class Beta1D(Fittable1DModel):
+    """
+    1-D Lorentz model with a varying power law, a.k.a, beta model.
+
+    Parameters
+    ----------
+    amplitude : float
+        Amplitude at x=xpos.
+    beta : float
+        Beta index.
+    r0 : float
+        Core radius.
+    xpos : float
+        Offset from x=0.
+
+    Notes
+    -----
+    Model formula:
+
+        .. math:: f(x) = a * (1 + [(x - x_{pos})/r_0] ^ 2)^(-3 * \\beta + 1/2)
+
+    See Also
+    --------
+    Lorentz1D
```

Comment:
this causes the sphinx fail, use the full namespace as Lorentz1D is not defined in this file.